x/merkledb/trieview.go

```diff
@@ -663,7 +662,16 @@ func (t *trieView) compressNodePath(parent, node *node) error {
 			return err
 		}
 
-		nextNode, err := t.getNodeFromParent(node, node.getSingleChildPath())
+		// there is only one child, but we don't know the index
+		// "cycle" over the key/values to find the only child
+		var childEntry child
+		var childPath path
```

Comment:
mega nit - but since we have multiple vars being declared here I think we should use a var block